CMakeLists.txt

```diff
@@ -224,7 +224,12 @@ set(CPACK_RPM_PACKAGE_RELOCATABLE ON)
 set(CPACK_ARCHIVE_COMPONENT_INSTALL OFF)
 
 set(CPACK_DEBIAN_PACKAGE_NAME "irods-icommands")
-set(CPACK_DEBIAN_PACKAGE_DEPENDS "${IRODS_PACKAGE_DEPENDENCIES_STRING}, irods-runtime (= ${CPACK_DEBIAN_PACKAGE_VERSION}), libc6, libssl1.0.0")
+
+if (IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "debian")
+  set(CPACK_DEBIAN_PACKAGE_DEPENDS "${IRODS_PACKAGE_DEPENDENCIES_STRING}, irods-runtime (= ${CPACK_DEBIAN_PACKAGE_VERSION}), libc6, libssl1.1")
+else()
+  set(CPACK_DEBIAN_PACKAGE_DEPENDS "${IRODS_PACKAGE_DEPENDENCIES_STRING}, irods-runtime (= ${CPACK_DEBIAN_PACKAGE_VERSION}), libc6, libssl1.0.0")
+endif()
```

Comment:
These lines can be collapsed to just line 229.

The if-statements aren't needed. All all debian-based OS supported by iRODS 4.3 provide `libssl1.1`.